.mergify.yml

```diff
@@ -2,6 +2,7 @@
 queue_rules:
   - name: default
     queue_conditions:
+      - status-success=^check-author-signed-cla
```

Comment:
It's missing the `~` to work as expected (regex). See https://github.com/crate/crate/pull/17207#issuecomment-2582634302
```Suggestion
      - status-success~=^check-author-signed-cla
```

Reply:
Ahh, good catch!